common/djangoapps/student/management/commands/bulk_change_enrollment_csv.py

```diff
@@ -0,0 +1,115 @@
+"""
+Management command to change many user enrollments in many courses using
+csv file.
+"""
+import csv
+import logging
+from os import path
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from student.models import CourseEnrollment, CourseEnrollmentAttribute, User
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class Command(BaseCommand):
+    """
+        Management command to change many user enrollments in many
+        courses using the csv file
+    """
+
+    help = """
+        Change the enrollment status of all the users specified in
+        the csv file in the specified course to specified course
+        mode.
+        Could be used to update effected users by order
+        placement issues. If large number of students are effected
+        in different courses.
+        Similar to bulk_change_enrollment but uses the csv file
+        input format and can enroll students in multiple courses.
+
+        Example:
+            $ ... bulk_change_enrollment_csv csv_file_path
+        """
+
+    def add_arguments(self, parser):
+        """ Add argument to the command parser. """
+        parser.add_argument(
+            '--csv_file_path',
+            required=True,
+            help='Csv file path'
+        )
+
+    def handle(self, *args, **options):
+        """ Main handler for the command."""
+        file_path = options['csv_file_path']
+
+        if not path.isfile(file_path):
+            raise CommandError("File not found.")
```

Comment:
We should also check that the format of the CSV file matches our expectations before moving forward here. Check that the columns are what we expect them to be.

Reply:
done